src/storage/v2/property_store.cpp

```diff
@@ -27,9 +29,13 @@
 #include "storage/v2/property_value.hpp"
 #include "storage/v2/temporal.hpp"
 #include "utils/cast.hpp"
+#include "utils/compressor.hpp"
 #include "utils/logging.hpp"
 #include "utils/temporal.hpp"
 
+// NOLINTNEXTLINE (cppcoreguidelines-avoid-non-const-global-variables)
+DEFINE_bool(property_store_compression, false, "Enable property store compression.");
```

Comment:
@DavIvek to be consistent with the rest of the flags this flags should be prefixed with the `storage` because it's part of the storage + because it's bool the `enabled` has to be there, so `--storage-property-store-compression-enabled` is the right flag name 😄

Reply:
@gitbuda Thanks for the reminder!